priv/repo/migrations/20220615023053_create_search_results.exs

```diff
@@ -0,0 +1,18 @@
+defmodule ElixirInternalCertificate.Repo.Migrations.CreateSearchResults do
+  use Ecto.Migration
+
+  def change do
+    create table(:search_results) do
+      add :top_ad_words_total, :integer
+      add :ad_words_total, :integer
+      add :non_ad_words_total, :integer
+      add :links_total, :integer
+      add :preview, :text
```

Comment:
Is is the raw html, right? then later we will render it which is preview. So how about to renaming it?

may be `html` or `html_response` instead of `preview `?